python/2d_structured_hllc_roe.py

```diff
@@ -0,0 +1,1174 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D
+import sys#,os
+# sys.path.append('../')
+from exact_riemann import wetbed
+from scipy.optimize import fsolve
+from scipy.integrate import solve_ivp
+
+
+C=0.5
+n = 0.0 #manning friction
+tend = 60.0
+g = 9.81;
+htol = 1.0e-2 #height tolerance below which hl&hr ; ul&ur is set to zero
+lx1 = -2.5; lx2 = 2.5;
+ly1 = -2.5; ly2 = 2.5;
+Nx = 100; #cells x-axis
+Ny = 100; #cells y-axis
+
+# bslope = ['flatbed','slope_upwards','slope_downwards','obstruction',
+#           'bump','flatbed+sloping_stage','2d_parabolic_dam','riemann_problem',
+#           '2d_dambreak','circular_dambreak','bump_steady_flow','bump_partial_cover']
+
+# choose the problem you need from the list of bslope below by setting ibslope
+# to the correct index
+bslope = ['bump','2d_parabolic_dam','riemann_problem',
+          '2d_dambreak','circular_dambreak','bump_steady_flow','bump_partial_cover']
+ibslope = 4

```

Comment:
I suggest to pass the option from command line arguments. This also applies to the following ibump, ibc,.... I can change it after the code is merged.